Include/internal/pycore_stackref.h

```diff
@@ -146,6 +144,16 @@ PyStackRef_CLOSE(_PyStackRef ref)
     Py_DECREF(obj);
 }
 
+static inline void
+PyStackRef_XCLOSE(_PyStackRef ref)
```

Comment:
Please update the documentation at the top of this file.

Reply:
This isn't new, it just needed to be moved.